provision/acc_provision/flavors.yaml

```diff
@@ -173,6 +173,10 @@ flavors:
     default_version: 1.9
     status: Experimental
     hidden: false
+    config:
+      aci_config:
+          vmm_domain:
+            type: DockerUCP
```

Comment:
On the APIC side we don't have a DockerUCP VMM domain. So I am wondering why this was added? Would it the VMM domain not default to Kubernetes if we don't add this? I believe the intention is to make this available as part of Kubernetes VMM domain.

Reply:
i dont think so. DockerUCP is different than K8s. So I have added new VVM_type for it.

Reply:
We are not creating a new VMM for DockerUCP.